### PR TITLE
Productionize cognito branch in #69

### PR DESCRIPTION
**Pull request by tamsky**
This PR is vs @goruha 's  PR #69.
It contains changes requested via code review.

### CODE REVIEW

File: releases/aws-alb-ingress-controller.yaml [hunk 1]
```diff
@@ -92,12 +96,15 @@ releases:
...
       # resources, such as Minikube. If you do want to specify resources, uncomment the following
       # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-      # limits:
-      #  cpu: 100m
-      #  memory: 128Mi
-      # requests:
-      #  cpu: 100m
-      #  memory: 128Mi
+      #
+      # TODO(tamsky): Should this ^^^ comment disappear ?
```
Comment (osterman) [inline releases/aws-alb-ingress-controller.yaml]: Lets parameterize it like the others. Refer to the other releases for examples

File: releases/aws-alb-ingress-controller.yaml [hunk 2]
```diff
@@ -27,13 +27,22 @@ releases:
...
   wait: true
   values:
-  - clusterName: us-west-2.testing.cloudposse.co
+  ### Required: AWS_ALB_INGRESS_CONTROLLER_KUBERNETES_CLUSTER_NAME;
+  ###           e.g. "us-west-2.testing.cloudposse.co"
+  - rbac:
+      ## If true, create & use RBAC resources
+      ##
+      create: '{{ env "RBAC_ENABLED" | default "false" }}'
+      serviceAccountName: default
```
Comment (goruha) [inline releases/aws-alb-ingress-controller.yaml]:
```suggestion
      serviceAccountName: `{{ env "AWS_ALB_INGRESS_CONTROLLER_SERVICE_ACCOUNT_NAME" | default "default" }}`
```

**Review by goruha — CHANGES_REQUESTED** [review]

File: releases/aws-alb-ingress-controller.yaml (hunk 1 above)
Comment (tamsky) [reply releases/aws-alb-ingress-controller.yaml]: Removed.

File: releases/aws-alb-ingress-controller.yaml (hunk 2 above)
Comment (tamsky) [reply releases/aws-alb-ingress-controller.yaml]: Thanks!
Fixed.

**Pull request merged**